tests/integration/targets/role_check_local_time/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- environment:
+    SC_HOST: "{{ sc_host }}"
+    SC_USERNAME: "{{ sc_username }}"
+    SC_PASSWORD: "{{ sc_password }}"
+    SC_TIMEOUT: "{{ sc_timeout }}"
+
+  block:
+    - name: Retrieve info about Time Zone
+      scale_computing.hypercore.time_zone_info:
+      register: time_zone
+
+    - name: Check if local time meets required time interval
+      ansible.builtin.include_role:
+        name: scale_computing.hypercore.check_local_time
+      vars:
+        time_zone: time_zone.record.zone
+        time_interval: 11:00-1:30 # Sometimes will pass, sometimes will not
```

Comment:
An optional param (default is "UTC now") containing seconds since epoch to explicitly set current time. it will allow testing whole thing.

Reply:
@justinc1 In unit tests I did some refactoring, so I was able to write the tests. In integration test I solved this problem with using ansible_date_time variable (it's not perfect, but I think is better than implementing optional param just for testing). But now I have different problem - how to register the result from the role - I would like to assert the success or fail message but have no idea how to do that - any tip for that? :)